docs/resources/Auto_Moderation.md

```diff
@@ -59,25 +59,26 @@ Rules can be configured to automatically execute actions whenever they trigger.
 ###### Trigger Types
 Characterizes the type of content which can trigger the rule.
 
-| Trigger Type   | Value | Description                                                          | Max per Guild |
-| -------------- | ----- | -------------------------------------------------------------------- | ------------- |
-| KEYWORD        | 1     | check if content contains words from a user defined list of keywords | 6             |
-| SPAM           | 3     | check if content represents generic spam                             | 1             |
-| KEYWORD_PRESET | 4     | check if content contains words from internal pre-defined wordsets   | 1             |
-| MENTION_SPAM   | 5     | check if content contains more unique mentions than allowed          | 1             |
+| Trigger Type   | Value | Description                                                                 | Max per Guild |
+| -------------- | ----- | --------------------------------------------------------------------------- | ------------- |
+| KEYWORD        | 1     | check if content contains words from a user defined list of keywords        | 6             |
+| SPAM           | 3     | check if content represents generic spam                                    | 1             |
+| KEYWORD_PRESET | 4     | check if content contains words from internal pre-defined wordsets          | 1             |
+| MENTION_SPAM   | 5     | check if content contains more unique mentions than allowed                 | 1             |
+| MEMBER_PROFILE | 6     | check if member profile contains words from a user defined list of keywords | 1             |
```

Comment:
Is it also possible to create a rule with trigger type `KEYWORD_PRESET` and event type `MEMBER_UPDATE`?

Reply:
> Is it also possible to create a rule with trigger type `KEYWORD_PRESET` and event type `MEMBER_UPDATE`?

i dont think so
![image](https://user-images.githubusercontent.com/76836550/229373819-5ccbcf3f-ad2e-4068-9e10-b66aebdc8b61.png)